secant/Features/BackupFlow/RecoveryPhraseValidation.swift

```diff
@@ -0,0 +1,229 @@
+//
+//  RecoveryPhraseValidation.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 10/29/21.
+//
+
+import Foundation
+import ComposableArchitecture
+
+typealias RecoveryPhraseValidationStore = Store<RecoveryPhraseValidationState, RecoveryPhraseValidationAction>
+typealias RecoveryPhraseValidationViewStore = ViewStore<RecoveryPhraseValidationState, RecoveryPhraseValidationAction>
+
+struct RecoveryPhraseEnvironment {
+    var mainQueue: AnySchedulerOf<DispatchQueue>
+    var newPhrase: () -> Effect<RecoveryPhrase, AppError>
+}
+
+/// Represents the completion of a group of recovery words by de addition of one word into the given group
+struct RecoveryPhraseStepCompletion: Equatable {
```

Comment:
Using the name `completion` here causes some readability issues with the notion of what most swift developers would think of as a "completion".  Usually that word is for describing a trailing closure or a callback that is to be executed upon completion of some sort of asynchronous code.  Is there another term we could use so that we don't muddy those naming waters?

Reply:
Yes, you are totally right. I may defer that to you both my dear native English speakers 😄

Reply:
According to the thesaurus, "fulfillment" is a synonym of "completion" so I renamed this to `RecoveryPhraseStepFulfillment` accordingly.